src/vaadin-radio-group-mixin.ts

```diff
@@ -60,37 +60,44 @@ export const RadioGroupMixin = <T extends Constructor<LitElement & KeyboardDirec
     /**
      * String used for the label element.
      */
-    @property({ type: String }) label: string | null | undefined;
+    @property({ type: String })
+    label: string | null | undefined;
 
     /**
      * Value of the radio group.
      */
-    @property({ type: String }) value: string | null | undefined;
+    @property({ type: String })
+    value: string | null | undefined;
 
     /**
      * When true, the radio group value is invalid.
      */
-    @property({ type: Boolean, reflect: true }) invalid: boolean | null | undefined = false;
+    @property({ type: Boolean, reflect: true })
+    invalid = false;
 
     /**
      * Radio group is aligned vertically by default. Set to "horizontal" to change this.
      */
-    @property({ type: String, reflect: true }) orientation: string | null | undefined;
+    @property({ type: String, reflect: true })
+    orientation: string | null | undefined;
 
     /**
      * When true, the radio group must have a value set by the user.
      */
-    @property({ type: Boolean, reflect: true }) required: boolean | null | undefined;
+    @property({ type: Boolean, reflect: true })
+    required = false;
 
     /**
      * When true, the user cannot modify the value of the radio group.
      */
-    @property({ type: Boolean, reflect: true }) readonly: boolean | null | undefined;
+    @property({ type: Boolean, reflect: true, attribute: 'readonly' })
+    readOnly?: boolean;
```

Comment:
Is there some reason why this one isn't like the other boolean props?
```suggestion
    @property({ type: Boolean, reflect: true, attribute: 'readonly' })
    readOnly = false;
```
And same for the `RadioGroupInterface` above (where it is `readOnly?: boolean;` with the question mark).

Reply:
Done.